### PR TITLE
fix methods

### PR DESCRIPTION
**Pull request by qiaozha**

### CODE REVIEW

File: src/plugins/azgenerator/CodeModelAzImpl.ts [hunk 1]
```diff
@@ -9,6 +9,13 @@ import { serialize, deserialize } from "@azure-tools/codegen";
 import { Session, startSession, Host, Channel } from '@azure-tools/autorest-extension-base';
 import { ToSnakeCase } from '../../utils/helper';
 
+function sortOperationByAzCommand(model: CodeModel) {
```
Comment (zikalino) [inline src/plugins/azgenerator/CodeModelAzImpl.ts]: just make it a private metod of CodeModelCliImpl

File: src/plugins/azgenerator/CodeModelAzImpl.ts [hunk 2]
```diff
@@ -422,12 +491,12 @@ export class CodeModelCliImpl implements CodeModelAz
 
     public get MethodParameter_Name(): string
     {
-        return ToSnakeCase(this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentOperationIndex].request.parameters[this.currentParameterIndex].language['az'].name);
+        return ToSnakeCase(this.codeModel.operationGroups[this.currentOperationGroupIndex].operations[this.currentMethodIndex].request.parameters[this.currentParameterIndex].language['az'].name);
```
Comment (zikalino) [inline src/plugins/azgenerator/CodeModelAzImpl.ts]: this should really come from "python", but as python namer is not integrated yet, i will add an issue to change that later

File: src/plugins/azgenerator/CodeModelAzImpl.ts (hunk 1 above)
Comment (qiaozha) [reply src/plugins/azgenerator/CodeModelAzImpl.ts]: changed now

File: src/plugins/azgenerator/CodeModelAzImpl.ts (hunk 2 above)
Comment (qiaozha) [reply src/plugins/azgenerator/CodeModelAzImpl.ts]: okay

**Pull request merged**